dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -268,8 +268,16 @@
           # Lower values will ensure that failed or slow API calls don't block other backups but may increase the risk
           # that backups can't be stored if uploading parts of the backup takes longer than the configured timeout.
           # See https://github.com/aws/aws-sdk-java-v2/blob/master/docs/BestPractices.md#utilize-timeout-configurations
+          # This setting can also be overridden using the environment variable ZEEBE_BROKER_DATA_BACKUP_S3_APICALLTIMEOUT
           # apiCallTimeout: PT180S
 
+          # When enabled, forces the s3 client to use path-style access. By default, the client will automatically
+          # choose between path-style and virtual-hosted-style. Should only be enabled if the s3 compatible storage cannot
```

Comment:
:wrench: `choose between path-style and virtual-hosted-style`? By default it uses virtual-hosted-style, right? This sentence is confusing.

Reply:
It's not guaranteed! To quote from the changelog:

> the SDK will use virtual bucket addressing as long as the bucket is a valid DNS label

I'm assuming AWS might change this in the future so I didn't want to guarantee anything in the docs and just leave it open which one will be used.

Reply:
:+1: